spec/tree_spec.rb

```diff
@@ -2,13 +2,75 @@
 require_relative '../lib/tree'
 
 describe Tree do
+
+  let(:tree) {Tree.new}
+
   it 'should be a Class' do
-    expect(described_class.is_a? Class).to eq true
+    expect(Tree.is_a? Class).to eq true
+  end
+
+
+  it 'should start with nothing' do
+    expect(tree.age).to eq 0
+    expect(tree.height).to eq 0
+    expect(tree.apples.size).to eq 0
+    expect(tree.alive).to eq true
+  end
+
+  it "should be growing" do
+    tree.age!
+    expect(tree.age! > 0).to eq true
```

Comment:
`expect(tree.age > 0).to eq true` would make more sense here. The return value from `age!` is moot.